src/Diagnostics.ModelsAndUtils/Models/DataProviderMetadataQuery.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Diagnostics.ModelsAndUtils.Models
+{
+    public class DataProviderMetadataQuery
+    {
+        public string Text;
+        public string Url;
+        public string OperationName;
+    }
+}
```

Comment:
Should you use this class in KustoDataProvider too? https://github.com/Azure/Azure-AppServices-Diagnostics/blob/master2/src/Diagnostics.DataProviders/DataProviders/KustoDataProvider.cs#L113

Reply:
I wanted to but unfortunately that will break the complied detectors today that are using the `GetKustoQueryAsync `method.